sentry/src/main/java/io/sentry/TransactionOptions.java

```diff
@@ -0,0 +1,74 @@
+package io.sentry;
+
+import java.util.Date;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
+public final class TransactionOptions {
+
+  private @Nullable CustomSamplingContext customSamplingContext = null;
+  private boolean isBindToScope = false;
```

Comment:
```suggestion
  private boolean bindToScope = false;
```

The getter should have the `is`

Reply:
updated